src/test/java/org/takes/misc/VerboseListTest.java

```diff
@@ -0,0 +1,407 @@
+        final List<Object> collection = Collections.emptyList();
+        this.list.removeAll(collection);
+        Mockito.verify(this.origin).removeAll(collection);
+    }
+
+    /**
+     * VerboseList should delegate retainAll method to decorated List.
+     * @throws Exception If something goes wrong.
+     */
+    @Test
+    public final void delegatesRetainAll() throws Exception {
+        final List<Object> collection = Collections.emptyList();
+        this.list.retainAll(collection);
+        Mockito.verify(this.origin).retainAll(collection);
+    }
+
+    /**
+     * VerboseList should delegate clear method to decorated List.
+     * @throws Exception If something goes wrong.
+     */
+    @Test
+    public final void delegatesClear() throws Exception {
+        this.list.clear();
+        Mockito.verify(this.origin).clear();
+    }
+
+    /**
+     * VerboseList should delegate get method to decorated List.
+     * @throws Exception If something goes wrong.
+     */
+    @Test
+    public final void delegatesGet() throws Exception {
+        final int index = 5;
+        this.list.get(index);
+        Mockito.verify(this.origin).get(index);
+    }
+
+    /**
+     * VerboseList should delegate set method to decorated List.
+     * @throws Exception If something goes wrong.
+     */
+    @Test
+    public final void delegatesSet() throws Exception {
+        final int index = 5;
+        final Object obj = new Object();
+        this.list.set(index, obj);
+        Mockito.verify(this.origin).set(index, obj);
+    }
+
+    /**
+     * VerboseList should delegate indexOf method to decorated List.
+     * @throws Exception If something goes wrong.
+     */
+    @Test
+    public final void delegatesIndexOf() throws Exception {
+        final Object obj = new Object();
+        this.list.indexOf(obj);
+        Mockito.verify(this.origin).indexOf(obj);
+    }
+
+    /**
+     * VerboseList should delegate lastIndexOf method to decorated List.
+     * @throws Exception If something goes wrong.
+     */
+    @Test
+    public final void delegatesLastIndexOf() throws Exception {
+        final Object obj = new Object();
+        this.list.lastIndexOf(obj);
+        Mockito.verify(this.origin).lastIndexOf(obj);
+    }
+
+    /**
+     * VerboseList should delegate listIterator method to decorated List.
+     * @throws Exception If something goes wrong.
+     */
+    @Test
+    public final void delegatesListIterator() throws Exception {
+        this.list.listIterator();
+        Mockito.verify(this.origin).listIterator();
+        final int index = 5;
+        this.list.listIterator(index);
+        Mockito.verify(this.origin).listIterator(index);
+    }
+
+    /**
+     * VerboseList should delegate subList method to decorated List.
+     * @throws Exception If something goes wrong.
+     */
+    @Test
+    public final void delegatesSubList() throws Exception {
+        final int fridx = 3;
+        final int toidx = 5;
```

Comment:
@Happy-Neko Let's call this `to`, [avoid compound names](http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html).